bin/micro.js

```diff
@@ -101,7 +100,12 @@ async function start() {
     process.exit(1)
   })
 
-  server.listen(flags.port, flags.host, () => {
+  const listenArgs = [flags.port || 0]
```

Comment:
How would `port` be falsy if we default it to 3000?

Reply:
@TooTallNate empty string. I expected the option parser to accept an empty string as 'valid' and not use it as an excuse to fallback to default (where, instead, it should only populate the default if the `-p` option was never encountered at all).

Reply:
e.g. `micro -p ''`